esp_idf/spotflow/README.md

```diff
@@ -0,0 +1,97 @@
+# Spotflow Observability IOT Devices
+
+Device Component for Spotflow embedded observability platform.
+
+This Component provides a set of tools and libraries for [ESP IDF]() to
+send your logs to the Spotflow observability platform.
+
+It is integrated with ESP IDF as a component which utilizes the ESP IDF logging component to seamlessly integrate with current firmwares.
+
+Our solution was tested on the following Espressif boards (more are coming soon):
+* [ESP32-S3]()
+* [ESP32-C3]()
+* [ESP32-C6]()
+
+We currently support:
+* ESP IDF v5.x.x
+* Zephyr 4.1.x, 4.2.x, and 4.3.x
+* nRF Connect SDK 3.0.x and 3.1.x
+
+For more information, check [Spotflow](https://app.spotflow.io/) or [Spotflow documentation](https://docs.spotflow.io/).
+
+## Getting Started
+
+Register and get your Ingest key at [Spotflow](https://spotflow.io/signup).
+
+Follow the Quickstart guide that is available in our portal after registration.
+
+Alternatively, you can check example applications in the [examples]().
```

Comment:
Missing link. I suppose you know about it, adding the comment just to keep track.